integrated_channels/integrated_channel/migrations/0012_alter_contentmetadataitemtransmission_unique_together.py

```diff
@@ -0,0 +1,17 @@
+# Generated by Django 3.2.12 on 2022-04-05 20:50
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('integrated_channel', '0011_contentmetadataitemtransmission_plugin_configuration_id'),
+    ]
+
+    operations = [
+        migrations.AlterUniqueTogether(
+            name='contentmetadataitemtransmission',
+            unique_together=set(),
```

Comment:
interesting, does this actually do anything in the table?

Reply:
I looked into this and meant to comment on the PR. Apparently this is a known "thing" / "issue" with the django ORM when you're dropping a unique together. Ive seen the same empty-set statement in other PRs in our code base. It also ran fine on my machine and in CI.